packages/@atjson/offset-annotations/src/utils/convert-html-tables.ts

```diff
@@ -215,8 +215,6 @@ export function convertHTMLTablesToDataSet(
     let dataSet = new DataSet({
       ...table,
       id: undefined,
-      start: table.start,
-      end: table.start,
```

Comment:
Ahh, so a "fix" here would be to make the dataset a block with a length of 1 covering a parse annotation. That would produce the same annotations. I do agree that we should fix this serialization, but in the end, moving to a blocks / marks internal structure would make this bug go completely away.